submissions/mustbefail/dom-api-task/styles/main.css

```diff
@@ -0,0 +1,60 @@
+main {
+  border: 1px solid #eee;
+  padding: 15px;
+  display: block;
+  margin: 5px 5px 5px 320px;
+  height: 100%;
+}
+
+main h2 {
+  font-weight: 500;
+}
+
+main .cards-container {
+  display: flex;
+  flex-flow: row wrap;
```

Comment:
It's not a blocker, but you can use just `flex-wrap: wrap;` because `flex` has `flex-direction: row` by default